### PR TITLE
Changes to calibre companion - part two

### PR DESCRIPTION
**Pull request by robert00s**
Continue work on #2851
Close: #2851

### CODE REVIEW

**Comment by KenMaltby** [comment]
Before the "Plugin Manager", I thought that another term was used.  Besides it causing concerns with regards to the original Android Calibre Companion people, it has only a small subset of the functionality of the commercial software.  This may disappoint users expecting it to have all the functions and the same UI.

**Comment by KenMaltby** [comment]
"Caliber wireless connection" fits more with the caliber side of the issue, also.

**Comment by NiLuJe** [comment]
IIRC, the calibre component is called wireless driver, so wireless
connection or wireless client on our end feels right?

**Comment by Frenzie** [comment]
NB #2851

**Comment by KenMaltby** [comment]
I don't remember "calibre sync" making it into use, besides I think that "calibre wireless connection" both fits better with the caliber terminology and the function being referenced.  It is not actually performing a sync., is it?

**Comment by Frenzie** [comment]
I think the current name (wireless connection) is perfectly fine. That's just to provide all relevant history.

**Comment by Frenzie** [comment]
tl;dr no name changes please

**Comment by robert00s** [comment]
The old name is left (calibre wireless connection).
I think is RTM.
